frontend/src/pages/contacts/ContactStatusSelect.tsx

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { SelectFormControl } from '../../components/molecules/formControls/selectFormControl';
+import { uri } from '../../utils';
+
+interface Props {
+	scoreDigit: any;
+}
```

Comment:
The `scoreDigit` type should be relative to the function. Don't use the `any` type for functions.
You could use a type like this for it:
```js
interface Props {
    scoreDigit(results: any): void;
}
```
You could also make sure about the `results` type too and use the relative type for it if there is any.